### ISSUE TITLE
other: POSTMASTER_ADDRESS in 15-lda.conf

### ISSUE THREAD

**Issue by jsonn** [tomav/docker-mailserver#3850]
### Subject

Something else that requires developers attention

### Description

In the main Dockerfile (https://github.com/docker-mailserver/docker-mailserver/blob/2018be7fdc04ce313667d365427fa9e79b4db2eb/Dockerfile#L99) 15-lda.conf is currently patched to contain the value of POSTMASTER_ADDRESS with a fallback value. It's not clear to me how this is supposed to ever work:

1. The environment variable is not set at this point in the build environment.
2. The line is replaced with the correct runtime value anyway (https://github.com/docker-mailserver/docker-mailserver/blob/2018be7fdc04ce313667d365427fa9e79b4db2eb/target/scripts/startup/setup.d/dovecot.sh#L12)

Is there any point left in doing the change in the main Dockerfile?

**Comment by polarathene** [comment]
> Is there any point left in doing the change in the main Dockerfile?

No, it should be removed. v14 changed Debian image and carries a bunch of other changes, so refactoring the `Dockerfile` has been delayed to a release after that.

There's a lot in there that can be pulled out into a separate build script like we do with `packages.sh` and `compile.sh`.

Regarding postmaster, it has a variety of concerns to address: https://github.com/docker-mailserver/docker-mailserver/issues/3769
